conf/gear.conf

```diff
@@ -186,7 +186,7 @@ gearpump {
   ###########################
   yarn {
     client {
-      package-path = "/user/gearpump/gearpump.zip"
+      package-path = "/usr/lib/gearpump/gearpump-${version}.zip"
```

Comment:
wrong format "gearpump-${version}.zip"

It means you have to create a file named gearpump-${version}.zip